rclcpp/executors/cbg_executor/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>examples_rclcpp_cbg_executor</name>
+  <version>0.0.1</version>
```

Comment:
I am actually not completely sure about this, but I believe all packages within the same repo have to have the same version number.
```suggestion
  <version>0.10.2</version>
```

Reply:
Yes, that's correct, they all have to have the same version number.